scipy/special/orthogonal.py

```diff
@@ -262,7 +264,10 @@ def roots_jacobi(n, alpha, beta, mu=False):
     if alpha == beta:
         return roots_gegenbauer(m, alpha+0.5, mu)
 
-    mu0 = 2.0**(alpha+beta+1)*cephes.beta(alpha+1, beta+1)
+    if (alpha + beta) <= 1000:
```

Comment:
Certainly a better way to do it, though an even better way would be to use the Lanczos approximation on all the Gamma terms in the Beta function. You'd need to use this private function exposed in `_ufuncs`:

https://github.com/scipy/scipy/blob/master/scipy/special/functions.json#L38

It's ok if you don't want to do that.

Reply:
Ah, that's a cool idea -- since there's no rush for this to be patched I'm happy to try that. I'll try to push an update within the next few days

Reply:
@person142 So I went through testing a Lanczos approximation based solution in both places (here and in the other commented place) but I was getting lower accuracy than using the current proposed changes (around a 1e-7 relative error). I think that I might have accidentally neglected some numerical issues with how I was using Lanczos, so a more sophisticated implementation by a more clever developer could probably do better than what we have here, but it does make me a little concerned about the complexity and reduced portability/maintainability of such an implemnetation.

Since the proposed changes are accurate to a relative error of around 1e-13 even for very large values of alpha+beta, I'd like to suggest that we move forward with the current approach -- though I'd be happy to add a comment about the current approach and/or suggesting a switch to use the Lanczos approximation in the future.

Reply:
Yes, moving forward with this approach sounds good. It's still very clearly an improvement, which should be the bar for merging.